solutions/standard/variables.tf

```diff
@@ -354,3 +354,9 @@ variable "elasticsearch_full_version" {
   type        = string
   default     = null
 }
+
```

Comment:
Update the variable description for `elasticsearch_full_version` to mention that the value is only used if `enable_kibana_dashboard` is true, and if no value is passed for `kibana_image_reference`